packages/logs/src/domain/logger.spec.ts

```diff
@@ -94,6 +94,16 @@ describe('Logger', () => {
       })
     })
 
+    it('should be configurable to "console" and "http"', () => {
```

Comment:
I'd be in favor of adding a test for silent + console, just to document the expected behavior